web-components/src/components/blog-post/index.tsx

```diff
@@ -77,6 +75,7 @@ const BlogPost: React.FC<BlogPostProps> = ({
         )}
       </Typography>
       <OutlinedButton
+        size="small"
```

Comment:
this was a bit out of scope, but I was frustrated that our patterns weren't in line with some of MUI's suggestions - specifically with the button's `size` attribute. Before this PR, it didn't really affect anything but font size (and not on all breakpoints) - now `small`, `medium` and `large` render buttons in accordance with the figma styleguide, so we shouldn't have to pass so many custom styles to buttons. I'll point out a few relevant areas...